spec/factories/kudo.rb

```diff
@@ -4,5 +4,6 @@
     content { 'Nice work!' }
     association :giver, factory: :employee
     association :receiver, factory: :employee
+    association :company_value, factory: :company_value
```

Comment:
No need to specify it explicitly, if association and factory name are the same
```suggestion
    association :company_value
```